plugins/window_size/linux/window_size_plugin.cc

```diff
@@ -15,6 +15,7 @@
 
 #include <flutter_linux/flutter_linux.h>
 #include <gtk/gtk.h>
+#include <cstring>
```

Comment:
This project follows the Google C++ style guide; please use the [include structure described there](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes).